### PR TITLE
Add the Concord45 keyboard.

### PR DESCRIPTION
**Pull request by DigiD3ar**
## Description

Add the Concord45 keyboard.
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keeblounge/concord45/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+     * ├────┬──┴─┬─┴──┬┴───┴───┴───┴───┴───┴───│───┴┬───┘ │   │ └───┘
+     * │RSET│    │    │                        │    │ ┌───│───│───┐
+     * └────┴────┴────┴────────────────────────┴────┘ │   │   │   │
+     *                                                └───┴───┴───┘
+     */
+    [_LN] = LAYOUT(KC_PWR, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, XXXXXX, XXXXXX, XXXXXX, XXXXXX, KC_MPRV, KC_MNXT, KC_VOLU, KC_SLEP, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, XXXXXX, XXXXXX, XXXXXX, XXXXXX, KC_MPLY, KC_VOLD, KC_WAKE, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, RESET, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX),
+};
+void matrix_scan_user(void){
+    // Empty
+};
```
Comment (drashna) [inline keyboards/keeblounge/concord45/keymaps/default/keymap.c:60]:
```suggestion
```

File: keyboards/keeblounge/concord45/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,51 @@
+#include QMK_KEYBOARD_H
+
+#define XXXXXX KC_TRNS
```
Comment (drashna) [inline keyboards/keeblounge/concord45/keymaps/via/keymap.c:3]: These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```

File: keyboards/keeblounge/concord45/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+     * │RSET│    │    │                        │    │ ┌───│───│───┐
+     * └────┴────┴────┴────────────────────────┴────┘ │   │   │   │
+     *                                                └───┴───┴───┘
+     */
+    [2] = LAYOUT(KC_PWR, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, XXXXXX, XXXXXX, XXXXXX, XXXXXX, KC_MPRV, KC_MNXT, KC_VOLU, KC_SLEP, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, XXXXXX, XXXXXX, XXXXXX, XXXXXX, KC_MPLY, KC_VOLD, KC_WAKE, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, RESET, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX),
+    [3] = LAYOUT(XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX),
+};
+void matrix_scan_user(void){
+    // Empty
+};
```
Comment (drashna) [inline keyboards/keeblounge/concord45/keymaps/via/keymap.c:51]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]
Also, could you add a license header to all of the C and H files in this PR?

File: keyboards/keeblounge/concord45/keymaps/via/keymap.c (hunk 3 above)
Comment (DigiD3ar) [reply keyboards/keeblounge/concord45/keymaps/via/keymap.c:51]: should delete that method?

File: keyboards/keeblounge/concord45/config.h [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+#include "config_common.h" // incluye archivo de configuracion comun
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4B4C //KL
+#define PRODUCT_ID      0x4334 //K3
+#define DEVICE_VER      0x0001 //0001
+#define MANUFACTURER    Keeblounge
+#define PRODUCT         Concord45
+#define DESCRIPTION     50 percent staggered keyboard
```
Comment (fauxpark) [inline keyboards/keeblounge/concord45/config.h:28]:
```suggestion
```

File: keyboards/keeblounge/concord45/keymaps/via/rules.mk [hunk 5]
```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+LTD_ENABLE = yes
```
Comment (fauxpark) [inline keyboards/keeblounge/concord45/keymaps/via/rules.mk:2]:
```suggestion
LTO_ENABLE = yes
```

File: keyboards/keeblounge/concord45/readme.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# Concord45
+
+![Show](https://i.ibb.co/bbZjg7N/Whats-App-Image-2020-08-21-at-13-47-14.jpg)
+
+A compact 45% (13x4) staggered keyboard kit made and sold by Keeb Lounge .
+
+* Keyboard Maintainer: [Digid3ar](https://github.com/DigiD3ar)
+
+
```
Comment (fauxpark) [inline keyboards/keeblounge/concord45/readme.md:9]:
```suggestion
Make example for this keyboard (after setting up your build environment):

    make keeblounge/concord45:default
```

File: keyboards/keeblounge/concord45/rules.mk [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (fauxpark) [inline keyboards/keeblounge/concord45/rules.mk:11]:
```suggestion
```

File: keyboards/keeblounge/concord45/rules.mk [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+MIDI_ENABLE = no            # MIDI controls
+AUDIO_ENABLE = no           # Audio output on port C6
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = no       # Enable WS2812 RGB underlight.
```
Comment (fauxpark) [inline keyboards/keeblounge/concord45/rules.mk:29]:
```suggestion
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/keeblounge/concord45/keymaps/default/keymap.c [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+     * ├────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴───┼───┼───┤
+     * │ SLP │F7 │F8 │F9 │F10│F11│F12│   │   │   │   │ PLAY │   │VL-│
+     * ├─────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴──┬───┴───│───│
+     * │ WAKE  │   │   │   │   │   │   │   │   │   │    │ ┌───┐ │   │
+     * ├────┬──┴─┬─┴──┬┴───┴───┴───┴───┴───┴───│───┴┬───┘ │   │ └───┘
+     * │RSET│    │    │                        │    │ ┌───│───│───┐
+     * └────┴────┴────┴────────────────────────┴────┘ │   │   │   │
+     *                                                └───┴───┴───┘
+     */
+    [_LN] = LAYOUT(KC_PWR, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, XXXXXX, XXXXXX, XXXXXX, XXXXXX, KC_MPRV, KC_MNXT, KC_VOLU, KC_SLEP, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, XXXXXX, XXXXXX, XXXXXX, XXXXXX, KC_MPLY, KC_VOLD, KC_WAKE, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, RESET, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX),
```
Comment (noroadsleft) [inline keyboards/keeblounge/concord45/keymaps/default/keymap.c:56]:
```suggestion
    [_LN] = LAYOUT(
        KC_PWR,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_MPRV, KC_MNXT, KC_VOLU,
        KC_SLEP, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,          KC_MPLY, KC_VOLD,
        KC_WAKE, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,          XXXXXXX, XXXXXXX,
        RESET,   XXXXXXX, XXXXXXX,                   XXXXXXX, XXXXXXX,                                     XXXXXXX, XXXXXXX, XXXXXXX
    ),
```

File: keyboards/keeblounge/concord45/keymaps/default/keymap.c [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+     * ├───┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴───┤   ├───┤
+     * │    │ 1 │ 2 │ 3 │ 4 │ 5 │ 6 │ 7 │ 8 │ 9 │ 0 │      │   │End│
+     * ├────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴──┬───┘   ├───┤
+     * │      │ ( │ ) │ ; │ . │   │ - │ + │ * │ / │ =  │ ┌───┐ │   │
+     * ├────┬─┴─┬─┴──┬┴───┴───┴───┴───┴───┴───│───┴┬───┘ │   │ └───┘
+     * │    │   │    │                        │    │ ┌───┤───├───┐
+     * └────┴───┴────┴────────────────────────┴────┘ │   │   │   │
+     *                                               └───┴───┴───┘
+     */
+    [_FN] = LAYOUT(XXXXXX, KC_EXLM, KC_AT, KC_HASH, KC_DOLLAR, KC_PERCENT, KC_CIRC, KC_AMPR, KC_ASTR, KC_LBRC, KC_RBRC, KC_BSLS, XXXXXX, KC_HOME, XXXXXX, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_ENTER, KC_END, XXXXXX, KC_LPRN, KC_RPRN, KC_SCLN, KC_DOT, XXXXXX, KC_MINS, KC_PLUS, KC_ASTR, KC_SLSH, KC_EQL, XXXXXX, KC_NO, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX),
```
Comment (noroadsleft) [inline keyboards/keeblounge/concord45/keymaps/default/keymap.c:42]:
```suggestion
    [_FN] = LAYOUT(
        XXXXXXX, KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC, KC_CIRC, KC_AMPR, KC_ASTR, KC_LBRC, KC_RBRC, KC_BSLS, XXXXXXX, KC_HOME,
        XXXXXXX, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,             KC_ENT,  KC_END,
        XXXXXXX, KC_LPRN, KC_RPRN, KC_SCLN, KC_DOT,  XXXXXXX, KC_MINS, KC_PLUS, KC_ASTR, KC_SLSH, KC_EQL,           XXXXXXX, KC_NO,
        XXXXXXX, XXXXXXX, XXXXXXX,                   XXXXXXX, XXXXXXX,                                     XXXXXXX, XXXXXXX, XXXXXXX
    ),
```

File: keyboards/keeblounge/concord45/keymaps/default/readme.md [hunk 11]
```diff
@@ -0,0 +1,5 @@
+![Concord45](https://i.ibb.co/bbZjg7N/Whats-App-Image-2020-08-21-at-13-47-14.jpg)
+
+# Default Concord45 Layout
```
Comment (noroadsleft) [inline keyboards/keeblounge/concord45/keymaps/default/readme.md:3]: Switch these two lines around.

```suggestion
# Default Concord45 Layout

![Concord45](https://i.ibb.co/bbZjg7N/Whats-App-Image-2020-08-21-at-13-47-14.jpg)
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]
Also, the image used in your readme files shows 50 keys, but your code only defines 48. What's the story there?

GitHub tip: You can apply multiple suggestions to a single commit by using the [Files Changed tab](./files).

File: keyboards/keeblounge/concord45/keymaps/default/keymap.c (hunk 1 above)
Comment (DigiD3ar) [reply keyboards/keeblounge/concord45/keymaps/default/keymap.c]: added in last commit

File: keyboards/keeblounge/concord45/keymaps/via/keymap.c (hunk 2 above)
Comment (DigiD3ar) [reply keyboards/keeblounge/concord45/keymaps/via/keymap.c]: added in last commit

File: keyboards/keeblounge/concord45/keymaps/via/keymap.c (hunk 3 above)
Comment (DigiD3ar) [reply keyboards/keeblounge/concord45/keymaps/via/keymap.c:65]: added in last commit

File: keyboards/keeblounge/concord45/config.h (hunk 4 above)
Comment (DigiD3ar) [reply keyboards/keeblounge/concord45/config.h]: added in last commit

File: keyboards/keeblounge/concord45/keymaps/via/rules.mk (hunk 5 above)
Comment (DigiD3ar) [reply keyboards/keeblounge/concord45/keymaps/via/rules.mk]: added in last commit

File: keyboards/keeblounge/concord45/readme.md (hunk 6 above)
Comment (DigiD3ar) [reply keyboards/keeblounge/concord45/readme.md]: added in last commit

File: keyboards/keeblounge/concord45/rules.mk (hunk 7 above)
Comment (DigiD3ar) [reply keyboards/keeblounge/concord45/rules.mk]: added in last commit

File: keyboards/keeblounge/concord45/rules.mk (hunk 8 above)
Comment (DigiD3ar) [reply keyboards/keeblounge/concord45/rules.mk]: added in last commit

File: keyboards/keeblounge/concord45/keymaps/default/keymap.c [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+     * ├────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴───┼───┼───┤
+     * │ SLP │F7 │F8 │F9 │F10│F11│F12│   │   │   │   │ PLAY │   │VL-│
+     * ├─────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴──┬───┴───│───│
+     * │ WAKE  │   │   │   │   │   │   │   │   │   │    │ ┌───┐ │   │
+     * ├────┬──┴─┬─┴──┬┴───┴───┴───┴───┴───┴───│───┴┬───┘ │   │ └───┘
+     * │RSET│    │    │                        │    │ ┌───│───│───┐
+     * └────┴────┴────┴────────────────────────┴────┘ │   │   │   │
+     *                                                └───┴───┴───┘
+     */
+    [_LN] = LAYOUT(KC_PWR, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, XXXXXX, XXXXXX, XXXXXX, XXXXXX, KC_MPRV, KC_MNXT, KC_VOLU, KC_SLEP, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, XXXXXX, XXXXXX, XXXXXX, XXXXXX, KC_MPLY, KC_VOLD, KC_WAKE, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, RESET, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX, XXXXXX),
```
Comment (noroadsleft) [inline keyboards/keeblounge/concord45/keymaps/default/keymap.c:70]: Tidies up the block comments, and corrects the error. Your keymap is still using the `XXXXXX`, which is deleted, so the keymap doesn't compile.

```suggestion
    /* Default QWERTY layer
     * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┐   ┌───┐
     * │Esc│ Q │ W │ E │ R │ T │ Y │ U │ I │ O │ P │Del│BkS│   │PgU│
     * ├───┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴───┤   ├───┤
     * │Tab │ A │ S │ D │ F │ G │ H │ J │ K │ L │ ' │Enter │   │PgD│
     * ├────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴──┬───┘   ├───┤
     * │LShift│ Z │ X │ C │ V │ B │ N │ M │ , │ . │Sft/│ ┌───┐ │Fn │
     * ├────┬─┴─┬─┴──┬┴───┴───┴───┴───┴───┴───┼───┴┬───┘ │ ↑ │ └───┘
     * │LCtl│OS │LAlt│         Space          │ Ln │ ┌───┼───┼───┐
     * └────┴───┴────┴────────────────────────┴────┘ │ ← │ ↓ │ → │
     *                                               └───┴───┴───┘
     */
    [_BASE] = LAYOUT(
        KC_ESC,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_DEL,  KC_BSPC, KC_PGUP,
        KC_TAB,  KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_QUOT,          KC_ENT,  KC_PGDN,
        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_RSFT,          KC_UP,   MO(_FN),
        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                    MO(_LN),          KC_LEFT, KC_DOWN, KC_RGHT
    ),

    /* Main Numbers, Symbols & Function Layer (MOMENTARY)
     * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┐   ┌───┐
     * │   │ ! │ @ │ # │ $ │ % │ ^ │ & │ * │ [ │ ] │ \ │   │   │Hme│
     * ├───┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴───┤   ├───┤
     * │    │ 1 │ 2 │ 3 │ 4 │ 5 │ 6 │ 7 │ 8 │ 9 │ 0 │      │   │End│
     * ├────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴──┬───┘   ├───┤
     * │      │ ( │ ) │ ; │ . │   │ - │ + │ * │ / │ =  │ ┌───┐ │   │
     * ├────┬─┴─┬─┴──┬┴───┴───┴───┴───┴───┴───┼───┴┬───┘ │   │ └───┘
     * │    │   │    │                        │    │ ┌───┼───┼───┐
     * └────┴───┴────┴────────────────────────┴────┘ │   │   │   │
     *                                               └───┴───┴───┘
     */
    [_FN] = LAYOUT(
        XXXXXXX, KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC, KC_CIRC, KC_AMPR, KC_ASTR, KC_LBRC, KC_RBRC, KC_BSLS, XXXXXXX, KC_HOME,
        XXXXXXX, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,             KC_ENT,  KC_END,
        XXXXXXX, KC_LPRN, KC_RPRN, KC_SCLN, KC_DOT,  XXXXXXX, KC_MINS, KC_PLUS, KC_ASTR, KC_SLSH, KC_EQL,           XXXXXXX, XXXXXXX,
        XXXXXXX, XXXXXXX, XXXXXXX,                            XXXXXXX,                   XXXXXXX,          XXXXXXX, XXXXXXX, XXXXXXX
    ),

    /* ALTERNATE Function layer (MOMENTARY)
     * ┌────┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┐
     * │ PWR│F1 │F2 │F3 │F4 │F5 │F6 │   │   │   │   │PRV│NXT│   │VL+│
     * ├────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴───┼───┼───┤
     * │ SLP │F7 │F8 │F9 │F10│F11│F12│   │   │   │   │ PLAY │   │VL-│
     * ├─────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴──┬───┴───┼───┤
     * │ WAKE  │   │   │   │   │   │   │   │   │   │    │ ┌───┐ │   │
     * ├────┬──┴─┬─┴──┬┴───┴───┴───┴───┴───┴───┼───┴┬───┘ │   │ └───┘
     * │RSET│    │    │                        │    │ ┌───┼───┼───┐
     * └────┴────┴────┴────────────────────────┴────┘ │   │   │   │
     *                                                └───┴───┴───┘
     */
    [_LN] = LAYOUT(
        KC_PWR,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_MPRV, KC_MNXT, KC_VOLU,
        KC_SLEP, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,          KC_MPLY, KC_VOLD,
        KC_WAKE, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,          XXXXXXX, XXXXXXX,
        RESET,   XXXXXXX, XXXXXXX,                            XXXXXXX,                   XXXXXXX,          XXXXXXX, XXXXXXX, XXXXXXX
    ),

```

File: keyboards/keeblounge/concord45/keymaps/via/keymap.c [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
```
Comment (noroadsleft) [inline keyboards/keeblounge/concord45/keymaps/via/keymap.c:65]: Same deal here.

```suggestion
    /* Default QWERTY layer
     * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┐   ┌───┐
     * │Esc│ Q │ W │ E │ R │ T │ Y │ U │ I │ O │ P │Del│BkS│   │PgU│
     * ├───┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴───┤   ├───┤
     * │Tab │ A │ S │ D │ F │ G │ H │ J │ K │ L │ ' │Enter │   │PgD│
     * ├────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴──┬───┘   ├───┤
     * │LShift│ Z │ X │ C │ V │ B │ N │ M │ , │ . │Sft/│ ┌───┐ │Fn2│
     * ├────┬─┴─┬─┴──┬┴───┴───┴───┴───┴───┴───┼───┴┬───┘ │ ↑ │ └───┘
     * │LCtl│OS │LAlt│             Space      │Fn1 │ ┌───┼───┼───┐
     * └────┴───┴────┴────────────────────────┴────┘ │ ← │ ↓ │ → │
     *                                               └───┴───┴───┘
     */
    [0] = LAYOUT(
        KC_ESC,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_DEL,  KC_BSPC, KC_PGUP,
        KC_TAB,  KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_QUOT,          KC_ENT,  KC_PGDN,
        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  RSFT_T(KC_SLSH),  KC_UP,   MO(2),
        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                    MO(1),            KC_LEFT, KC_DOWN, KC_RGHT
    ),

    /* Main Numbers, Symbols & Function Layer (MOMENTARY)
     * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┐   ┌───┐
     * │   │ ! │ @ │ # │ $ │ % │ ^ │ & │ * │ [ │ ] │ \ │   │   │Hme│
     * ├───┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴───┤   ├───┤
     * │    │ 1 │ 2 │ 3 │ 4 │ 5 │ 6 │ 7 │ 8 │ 9 │ 0 │      │   │End│
     * ├────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴──┬───┘   ├───┤
     * │      │ ( │ ) │ ; │ . │   │ - │ + │ * │ / │ =  │ ┌───┐ │   │
     * ├────┬─┴─┬─┴──┬┴───┴───┴───┴───┴───┴───┼───┴┬───┘ │   │ └───┘
     * │    │   │    │                        │    │ ┌───┼───┼───┐
     * └────┴───┴────┴────────────────────────┴────┘ │   │   │   │
     *                                               └───┴───┴───┘
     */
    [1] = LAYOUT(
        XXXXXXX, KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC, KC_CIRC, KC_AMPR, KC_ASTR, KC_LBRC, KC_RBRC, KC_BSLS, XXXXXXX, KC_HOME,
        XXXXXXX, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,             KC_ENT,  KC_END,
        XXXXXXX, KC_LPRN, KC_RPRN, KC_SCLN, KC_DOT,  XXXXXXX, KC_MINS, KC_PLUS, KC_ASTR, KC_SLSH, KC_EQL,           XXXXXXX, XXXXXXX,
        XXXXXXX, XXXXXXX, XXXXXXX,                            XXXXXXX,                   XXXXXXX,          XXXXXXX, XXXXXXX, XXXXXXX
    ),

    /* ALTERNATE Function layer (MOMENTARY)
     * ┌────┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┐
     * │ PWR│F1 │F2 │F3 │F4 │F5 │F6 │   │   │   │   │PRV│NXT│   │VL+│
     * ├────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴───┼───┼───┤
     * │ SLP │F7 │F8 │F9 │F10│F11│F12│   │   │   │   │ PLAY │   │VL-│
     * ├─────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴──┬───┴───┼───┤
     * │ WAKE  │   │   │   │   │   │   │   │   │   │    │ ┌───┐ │   │
     * ├────┬──┴─┬─┴──┬┴───┴───┴───┴───┴───┴───┼───┴┬───┘ │   │ └───┘
     * │RSET│    │    │                        │    │ ┌───┼───┼───┐
     * └────┴────┴────┴────────────────────────┴────┘ │   │   │   │
     *                                                └───┴───┴───┘
     */
    [2] = LAYOUT(
        KC_PWR,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_MPRV, KC_MNXT, KC_VOLU,
        KC_SLEP, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,          KC_MPLY, KC_VOLD,
        KC_WAKE, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,          XXXXXXX, XXXXXXX,
        RESET,   XXXXXXX, XXXXXXX,                            XXXXXXX,                   XXXXXXX,          XXXXXXX, XXXXXXX, XXXXXXX
    ),
    [3] = LAYOUT(
        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,          XXXXXXX, XXXXXXX,
        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,          XXXXXXX, XXXXXXX,
        XXXXXXX, XXXXXXX, XXXXXXX,                            XXXXXXX,                   XXXXXXX,          XXXXXXX, XXXXXXX, XXXXXXX
    ),
};
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Pull request closed without merging**